console-api/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![doc = include_str!("../README.md")]
+#![allow(warnings)]
```

Comment:
hmm, could we, instead, add the attribute only to the modules that are generated by `prost`? the `common` module also contains some handwritten code: https://github.com/tokio-rs/console/blob/ff6d048e0464f55fc9d8202b05c7762bc99f9924/console-api/src/common.rs#L5-L240 and it would be nice if clippy could lint that. we might have to put the generated code in a submodule of `common`, like this:

```rust
// in `common.rs`

mod generated {
    #![allow(warnings)]
    include!("generated/rs.tokio.console.common.rs");
}

pub use self::generated::*;

// ...
```
for the other modules, we can just put the `#[allow(warnings)]` attribute on the declaration of those modules.

Reply:
Will do!

Reply:
Done.